parlai/crowdsourcing/tasks/acute_eval/example_script.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import os
+import time
+import shlex
+from mephisto.core.operator import Operator
+from parlai.crowdsourcing.tasks.acute_eval.acute_eval_blueprint import BLUEPRINT_TYPE
+from mephisto.utils.scripts import MephistoRunScriptParser
+
+"""
+Example script for running ACUTE-EVAL.
+The only argument that *must* be modified for this to be run is:
+``pairings_filepath``:  Path to pairings file in the format specified in the README.md
+
+The following args are useful to tweak to fit your specific needs;
+    - ``annotations_per_pair``: A useful arg if you'd like to evaluate a given conversation pair
+                                more than once.
+    - ``num_matchup_pairs``:    Essentially, how many pairs of conversations you would like to evaluate
+    - ``subtasks_per_unit``:     How many comparisons you'd like a turker to complete in one HIT
+
+"""
+
+TASK_DIRECTORY = os.path.dirname(os.path.realpath(__file__))
+
+parser = MephistoRunScriptParser()
+parser.add_argument(
+    "-pfp",
+    "--pairings-filepath",
+    default=f"{TASK_DIRECTORY}/pairings.jsonl",
+    help="Path to pairings file",
+)
+parser.add_argument(
+    "-app",
+    "--annotations-per-pair",
+    default=1,
+    help="Annotations per pairing, to ensure worker agreement, default 1",
+    type=int,
+)
+parser.add_argument(
+    "-nmp",
+    "--num-matchup-pairs",
+    default=2,
+    help="Number of pairs per model matchup, default 2",
+    type=int,
+)
+parser.add_argument(
```

Comment:
do we need those flags as they are already included in the blueprint?

Reply:
Having them here lets you set them from the command line. Hoping to make mephisto do this kind of thing automatically, but for now the transfer is manual.